From 8d7e2eaed10e2928358805f613223a6a149790d6 Mon Sep 17 00:00:00 2001
From: Hiago De Franco <hiago.franco@toradex.com>
Date: Wed, 31 Jul 2024 11:51:06 -0300
Subject: [PATCH] meson.build: Fix missing python_opt

'python_opt' was not declared earlier, giving the following build error:

| Message: python_abi_flags =
|
| ../gst-python-1.24.0/meson.build:84:16: ERROR: Unknown variable "python_opt".

Remove it and keep the error message.

Upstream-Status: Inappropriate [oe specific]

The patch already exists upstream, but this is related to another fix [1]
not suitable for OE. Moreover, the backport does not apply since
the patch was added to the gstreamer monorepo.

[1] https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/6734?

Signed-off-by: Hiago De Franco <hiago.franco@toradex.com>
---
 meson.build | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/meson.build b/meson.build
index f240b6fd8aca..53f7a43702c7 100644
--- a/meson.build
+++ b/meson.build
@@ -80,12 +80,7 @@ foreach loc: pylib_locs
   endforeach
 endforeach
 if pylib_fname == ''
-  error_msg = 'Could not find python library to load'
-  if python_opt.enabled()
-    error(error_msg)
-  else
-    message(error_msg)
-  endif
+  message('Could not find python library to load')
 endif
 
 pygi_override_dir = get_option('pygi-overrides-dir')
-- 
2.39.2

